### ISSUE TITLE
Scroll to New Comment

### ISSUE THREAD

**Issue by Sherlouk** [rnystrom/GitHawk#406]
Follow up from: https://github.com/rnystrom/GitHawk/issues/393#issuecomment-331669376

Also known as `MDEyOklzc3VlQ29tbWVudDMzMTY2OTM3Ni1IZWhlIEhpIQ==` 😉

**Comment by BasThomas** [comment]
Trying to tackle this one, but not sure if / how we can request that comment id for the notification?

**Comment by Sherlouk** [comment]
https://github.com/rnystrom/GitHawk/blob/master/Sample/_notifications.json#L72

Think it's the ID at the end of that line ^^

**Comment by BasThomas** [comment]
This would also allow us to link urls like https://github.com/rnystrom/GitHawk/pull/483#issuecomment-334179617 to a specific comment :o

**Comment by Sherlouk** [comment]
Who knew a simple base64 conversion would unlock so much functionality 😱

Probably just want a var that can be optionally defined when creating an IssueViewController which defines a comment to scroll to. Then that can be passed through from various places!

**Comment by BasThomas** [comment]
We don’t do this, do we?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Nope, doesn't seem like there's any demand, and I haven't found myself needing it
